src/main/java/com/salesforce/dev/pages/Home/HomePage.java

```diff
@@ -32,39 +34,8 @@ public HomePage(){
    }
 
    public LoginPage clickLoginBtn(){
-     wait.until(ExpectedConditions.visibilityOf(loginBtn));
-     loginBtn.click();
+     operation.clickWebElement(loginBtn);
      return new LoginPage(this.driver);
    }
 
-  /**
-   * Method that make sure to log with the correct user
-   *
-   * @author: Jimmy Vargas
-   * */
-
-  public MainPage loginAsPrimaryUser(){
```

Comment:
Keep in mind this method has a logic to keep the App's session
I mean if exist a session in the App you will use directly that session. you don't need to login again